tests/e2e/scale/noobaa/test_scale_obc_start_time_respin_noobaa_pods.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import time
+
+import pytest
+import csv
+
+from ocs_ci.ocs import constants, scale_noobaa_lib
+from ocs_ci.framework.testlib import scale, E2ETest
+from ocs_ci.helpers import helpers
+from ocs_ci.utility.utils import ocsci_log_path
+from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(autouse=True)
+def teardown(request):
+    def finalizer():
+        scale_noobaa_lib.cleanup(constants.OPENSHIFT_STORAGE_NAMESPACE)
+
+    request.addfinalizer(finalizer)
+
+
+@scale
+class TestScaleOBCStartTime(E2ETest):
+    """
```

Comment:
this docstrings not needed as you have this covered in the test function doc strings

Reply:
Fixed.